doc/benchmarking/benchmarking_tutorial.rst

```diff
@@ -107,9 +108,13 @@ Parameters of the BenchmarkExecutor Class
 
 This class creates a set of ``MotionPlanRequests`` that respect the parameters given in the supplied instance of ``BenchmarkOptions`` and then executes the requests on each of the planners specified.  From the ``BenchmarkOptions``, queries, ``goal_constraints``, and ``trajectory_constraints`` are treated as separate queries.  If a set of ``start_states`` is specified, each query, ``goal_constraint``, and ``trajectory_constraint`` is attempted with each start state (existing start states from a query are ignored).  Similarly, the (optional) set of path constraints is combined combinatorially with the start query and start ``goal_constraint`` pairs (existing ``path_constraint`` from a query are ignored).  The workspace, if specified, overrides any existing workspace parameters.
 
-The benchmarking pipeline does not utilize ``MoveGroup``, and ``PlanningRequestAdaptors`` are **not** invoked.
+The benchmarking pipeline does not utilize ``MoveGroup``.
+Instead, the planning pipelines are initialized and run directly including all specified ``PlanningRequestAdapters``.
+This is especially useful for benchmarking the effects of smoothing adapters.
 
-It is possible to customize a benchmark run by deriving a class from ``BenchmarkExecutor`` and overriding one or more of the virtual functions.  Additionally, a set of functions exists for ease of customization in derived classes:
+It is possible to customize a benchmark run by deriving a class from ``BenchmarkExecutor`` and overriding one or more of the virtual functions.
+For instance, overriding the functions ``initializeBenchmarks()`` or ``loadBenchmarkQueryData()`` allows to specify the benchmark queries directly and to provide a custom planning scene without using ROS warehouse.
```

Comment:
Providing pointers to example applications (e.g. in the unit tests) might be helpful.

Reply:
I added my own example from https://github.com/ros-planning/moveit/pull/1548

Reply:
I was even thinking of providing hyperlinks to the github code.